plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/artifacts/ManifestFetcher.kt

```diff
@@ -28,8 +29,12 @@ class ManifestFetcher(
         private val mapper = jacksonObjectMapper().apply { configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false) }
         private val logger = getLogger<ManifestFetcher>()
 
-        private const val DEFAULT_MANIFEST_URL =
-            "https://aws-toolkit-language-servers.amazonaws.com/qAgenticChatServer/0/manifest.json"
+        private val DEFAULT_MANIFEST_URL = getManifestEndpoint()
+
+        private fun getManifestEndpoint(): String {
+            val endpoint = Registry.get("amazon.q.flare.endpoint").asString()
+            return endpoint.ifBlank { "https://aws-toolkit-language-servers.amazonaws.com/qAgenticChatServer/0/manifest.json" }
+        }
```

Comment:
value will be read on class init
```suggestion
        private val DEFAULT_MANIFEST_URL
            get() = getManifestEndpoint()

        private fun getManifestEndpoint(): String {
            val endpoint = Registry.get("amazon.q.flare.endpoint").asString()
            return endpoint.ifBlank { "https://aws-toolkit-language-servers.amazonaws.com/qAgenticChatServer/0/manifest.json" }
        }
```
should probably just delete DEFAULT_MANIFEST_URL and only use the method